conda-lock.yml

```diff
@@ -0,0 +1,4591 @@
+# This lock file was generated by conda-lock (https://github.com/conda-incubator/conda-lock). DO NOT EDIT!
+#
+# A "lock file" contains a concrete list of package versions (with checksums) to be installed. Unlike
+# e.g. `conda env create`, the resulting environment will not change as new package versions become
+# available, unless you explicitly update the lock file.
+#
+# Install this environment as "YOURENV" with:
+#     conda-lock install -n YOURENV --file conda-lock.yml
```

Comment:
Official conda-lock instructions seems to be wrong here, had to remove the `--file`
```suggestion
#     conda-lock install -n YOURENV conda-lock.yml
```
Or for this repo, do `conda-lock install -n egu22pygmt conda-lock.yml`